src/Data/Product/Function/NonDependent/Propositional.agda

```diff
@@ -41,7 +48,7 @@ private
 ------------------------------------------------------------------------
 -- Combinators for various function types
 
-infixr 2 _×-⇔_ _×-↣_ _×-↠_ _×-⤖_ _×-↩_ _×-↪_ _×-↔_
+infixr 2 _×-⟶_ _×-⇔_ _×-↣_ _×-↠_ _×-⤖_ _×-↩_ _×-↪_ _×-↔_
```

Comment:
This needs a CHANGELOG entry.

Reply:
Done, along with other addition to CHANGELOG